AsanPardakht.IPG/Services.cs

```diff
@@ -60,7 +60,34 @@ public async Task<GenerateTokenResponse> GenerateToken(GenerateTokenRequest data
             }
 
         }
+        /// <summary>
+        /// add default sharing
+        /// </summary>
+        /// <param name="data"></param>
+        /// <returns></returns>
+        /// <exception cref="InvalidIdentityException"></exception>
+        /// <exception cref="AsanPardakhtIPGException"></exception>
+        public async Task<GenerateTokenResponse> GenerateShareToken(GenerateTokenRequest data)
```

Comment:
Isn't it better to name it as follows?

```suggestion
        public async Task<GenerateTokenResponse> GenerateTokenWithDefaultSharing(GenerateTokenRequest data)
```

Reply:
well as same as the others . do not change the name to different name from official name. and yes i think your name is better